pydatalab/tests/server/test_auth.py

```diff
@@ -7,8 +7,8 @@ def test_allow_emails():
     assert _check_email_domain("test@example.org", ["example.org", "example2.org"])
     assert _check_email_domain("test@subdomain.example.org", ["example.org", "example2.org"])
     assert _check_email_domain("test@subdomain.example.org", ["subdomain.example.org"])
-    assert _check_email_domain("test@example2.org", [])
-    assert not _check_email_domain("test@example2.org", None)
+    assert not _check_email_domain("test@example2.org", [])
+    assert _check_email_domain("test@example2.org", None)
```

Comment:
I would consider this a bug previously; this only affected the public deployment